src/utils/paymentQRHelper.test.ts

```diff
@@ -48,71 +48,190 @@ describe("tests for findValueByKey", () => {
   });
 });
 
-describe("tests for getUpdatedTransactionsPaymentQRIdentifiers", () => {
-  let transactionsBeforeUpdate: Array<Transaction>;
+describe("tests for getUpdatedTransactionsPaymentQRIdentifiers for non-NETS", () => {
+  let paymentQRPayloadWithoutNETS: string;
+  let paymentQRIdentifierInput: IdentifierInput;
+  let merchantNameIdentifierInput: IdentifierInput;
+  let merchantIdIdentifierInput: IdentifierInput;
 
-  beforeEach(() => {
-    transactionsBeforeUpdate = [
+  beforeAll(() => {
+    paymentQRPayloadWithoutNETS =
+      "00020101021126520008com.grab0136c99990dc-ab89-442f-accb-1fd96dd4eff627330015sg.com.dash.www0110000000901628460010com.myfave0128https://myfave.com/qr/6ntdz329360009SG.AIRPAY0119936009180000000468751810007SG.SGQR01121809112DE3C7020701.000703065743700402010503105060400000708202012105204581253037025802SG5920UDDERS UPPER THOMSON6009Singapore63047AEC";
+    paymentQRIdentifierInput = {
+      label: "Payment QR",
+      scanButtonType: "QR",
+      validationRegex: "[\\s\\S]*",
+      textInputType: "PAYMENT_QR",
```

Comment:
can add one test case for non `PAYMENT_QR` type

Reply:
Looks like I missed out this simple test case, thanks a lot for suggesting this! Turns out I did not handle the case where `PAYMENT_QR` does not exist in `identifierInputs`...

Added this test case in [22432d8](https://github.com/rationally-app/mobile-application/pull/532/commits/22432d8333345540e92e04a73be248b2f2110d5a)!